src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -1711,7 +1709,7 @@ UniqueGenerator<int64_t> intGenerator(const Node& node, GeneratorArgs generatorA
  * @param node
  *   a top-level document value i.e. either a scalar or a `^RandomInt` value
  * @return
- *   either a `^RantomInt` generator (etc--see `intParsers`)
+ *   either a `^RandomDouble` generator (etc--see `intParsers`)
```

Comment:
Shouldn't the parenthetical comment be updated also. hopefully see `doubleParsers`

Reply:
Oops, forgot that one.